run-all.sh

```diff
@@ -0,0 +1,20 @@
+# Copyright 2020 Google LLC
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     https://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+declare -a algs=("SharedGlobal" "SharedMultiZone" "Local" "LocalWeighted" "LocalOpt" "LocalShared" "Original")
+
+for alg in ${algs[@]}; do
+    echo "Running $alg"
+    go run main.go -input=example/range-input.csv -alg=$alg -output=example/$alg-range-output.csv
```

Comment:
`open example/range-input.csv: no such file or directory`

Reply:
Probably want to add a check with bash for the existence of this file before starting this loop. If it doesn't exist you could exit and log that the python generation script needs to run first.

Reply:
Also as a sidebar, would probably be good to use a directory other than `example` here and/or add `range-input.csv` to `.gitignore`

Reply:
added